homework-1/css/style.css

```diff
@@ -0,0 +1,1220 @@
+  padding: 8px 0;
+  border-top: 2px solid #37E63E;
+}
+.footer__title {
+  font-size: 16px;
+  line-height: 36px;
+  color: #7F7F7F;
+}
+.menu-footer__list li + li {
+  padding-top: 10px;
+}
+.menu__link {
+  font-family: "Acme-Regular";
+  font-size: 12px;
+  line-height: 24px;
+  letter-spacing: 2px;
+  text-transform: uppercase;
+  color: #7F7F7F;
+}
+.phone {
+  margin-top: 4px;
+  margin-bottom: 10px;
+  font-size: 16px;
+  line-height: 24px;
+  color: #7F7F7F;
+  transition: 1s;
+}
+.phone:hover {
+  color: #37e63e;
+}
+address {
+  font-size: 12px;
+  line-height: 20px;
+  color: #7F7F7F;
+  width: 170px;
+}
+.link a {
+  font-family: "Aclonica-Regular";
+  font-size: 12px;
+  line-height: 20px;
+  color: #37E63E;
+  transition: 1s;
+}
+.link a:hover {
+  color: #7f7f7f;
+  text-decoration: underline;
+}
+.social {
+  padding-bottom: 24px;
+}
+.social__inner {
+  border-top: 1.5px solid #b2b2b2;
+}
+.social-list {
+  display: flex;
+  align-items: center;
+  justify-content: space-around;
+  padding-top: 29px;
+  padding-bottom: 14px;
+}
+svg .social-hover {
+  transition: 200ms;
+  transition-timing-function: ease-in-out;
+  -webkit-transition: 200ms;
+  -webkit-transition-timing-function: ease-in-out;
+}
+svg:hover .social-hover {
+  fill: #37E63E;
+}
+.mobile__img {
+  display: block;
+  margin: 0 auto;
+  height: 38px;
+}
+.dekstop__img {
+  display: none;
+}
+@media (min-width: 640px) {
+  .container {
+    padding: 0 30px;
+  }
+  .header__banner-submenu {
+    padding: 0 20px;
+    top: 188px;
+  }
+  .header__banner .submenu__list {
+    height: 60px;
+    max-width: 770px;
+    margin: 0 auto;
+  }
+}
+@media (min-width: 767px) {
+  .container {
+    padding: 0 40px;
+  }
+}
+.header.sticky-header .logo__link {
+  color: #b2b2b2;
+}
+.header {
+  background: transparent;
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  transition: 0.6s;
+  padding: 18px;
+  z-index: 10000;
+  padding: 18px 84px;
+}
+.header.sticky-header {
+  background: #fff;
+}
+.header .logo {
+  position: relative;
+  transition: 0.6;
+}
+.header ul {
+  position: relative;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+.header ul li {
+  position: relative;
+}
+.header ul li a {
+  position: relative;
+  margin: 0 15px;
+  color: #fff;
+  transition: 0.6s;
+}
+.header__banner {
+  position: relative;
+  width: 100%;
+  height: 500px;
+  background-image: linear-gradient(60deg, #21c3d8 20%, #13d2c2 40%, #01f17d 75%);
+}
+.header.sticky-header .menu__icon::before, .header.sticky-header .menu__icon::after, .header.sticky-header .menu__icon span {
+  background-color: #b2b2b2;
+}
+.header.sticky-header ul li a {
+  color: #b2b2b2;
+}
+.header.sticky-header .header__button-link a {
+  color: #b2b2b2;
+  border: 1px solid #b2b2b2;
+}
+#banner-submenu.sticky {
+  position: fixed;
+  z-index: 100;
+  width: 100%;
+}
+#banner-submenu {
```

Comment:
Try to avoid using 'id' it is heavy selector. Please use classes instead. 
It relates to such cases in whole file. Please fix it

Reply:
fixed a bug

Reply:
done